app/assets/stylesheets/components/common/_download_link.scss

```diff
@@ -0,0 +1,11 @@
+// For links that contain download attribute
+
+a[download] {
+  &:after {
+    @extend %icon;
+    @extend .icon--pdf;
```

Comment:
What if it's a download but not a PDF? It would be good to see the code that generates the markup (or the markup itself). Can we add a class as well as adding the download attribute?